utils/python/CIME/case_setup.py

```diff
@@ -139,7 +139,7 @@ def case_setup(caseroot, clean=False, test_mode=False, reset=False):
         logger.info("Some files have been saved to %s" % backup_dir)
 
         with open("CaseStatus", "a") as fd:
-            fd.write("case.setup --clean %s\n" % time.strftime("%Y-%m-%d %H:%M:%S"))
+            fd.write("%s case.setup --clean \n" % time.strftime("%Y-%m-%d %H:%M:%S"))
```

Comment:
I think that we should put the CaseStatus append function in CIME.utils and use it uniformly throughout the code.